README.md

```diff
@@ -23,14 +25,22 @@ Token Server is exposed as resource. So in this example the Token Server acts as
-| resource.gateway.tokenserver.clientSecret | The client secret of the oauth client acting as resource gateway, see [Resource Gateway config](https://docs.onegini.com/public/token-server/topics/general-app-config/resource-gateway/resource-gateway.html) |
-| resource.gateway.tokenserver.baseUri      | The token server base uri.                                                                                                                                                                                     |
-| device.api.serverRoot                     | The base uri of the resource server (Token Server in this example).                                                                                                                                            |
-| device.api.username                       | The basic authentication username used to access the api.                                                                                                                                                      |
-| device.api.password                       | The basic authentication password used to access the api.                                                                                                                                                      |
+Refer to the Spring
+Boot [Externalized Configuration](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config) for
+setting the properties.
+
+The following properties can be set for the Example Resource Gateway:
+
+| Property                                  | Default value | Example value               | Description
+|-------------------------------------------|---------------|-----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+| resource.gateway.tokenserver.baseUri      |               | http://localhost:7878/oauth | The base URI of the [Onegini Token Server](https://docs.onegini.com/msp/stable/token-server).
+| resource.gateway.tokenserver.clientId     |               |                             | The client id of the API client acting as resource gateway, see [Resource Gateway config](https://docs.onegini.com/msp/stable/token-server/topics/general-app-config/resource-gateway/resource-gateway.html).
+| resource.gateway.tokenserver.clientSecret |               |                             | The client secret of the API client acting as resource gateway.
+| device.api.serverRoot                     | The value of resource.gateway.tokenServer.baseUri | http://localhost:7878/oauth | The base uri of the resource server that returns a list of Devices. In this example, it is the base URI of the Onegini Token Server.
```

Comment:
I wonder whether we need two properties pointing at the Token Server 🤔  Now we have:
- `resource.gateway.tokenserver.baseUr`
- `device.api.serverRoot `
Are there scenarios where these two would point to different instances?

Reply:
One uses the value of the other so you only need to configure it once.
I think the property separation was done to make it more explicit that this RG connects to the TS for Token introspection and also utilizes one of its APIs as Resource Server.

Reply:
Ok, thanks for the clarification 👍